### ISSUE TITLE
System.Exception: Unknown framework when trying to run tests

### ISSUE THREAD

**Issue by Tino92** [PragmaticFlow/NBomber#148]
I think it is related to the warnings from the build log:
https://ci.appveyor.com/project/PragmaticFlowOrg/nbomber#L418

**Comment by Tino92** [comment]
The actual error message was hidden behind this code:
https://github.com/PragmaticFlow/NBomber/blob/fa7c826c628c6d9d127c241e1e35f00ae4c6dcb4/src/NBomber/Infra/TestFrameworkRunner.fs#L31

I think "Unknown framework" is not a good enough error message. We are missing key details in the error message. What framework are you talking about? Why can't you just print the error message to console, even though you cannot recognise the test framework?

The underlying issue was: [14:25:23 ERR] Step names are not unique: 'pause'.
Since you removed Step.createPause, I was using step.Create() to create a task that delays for a given number of seconds, to emulate the functionality from createPause().

Why did you remove the createPause() functionality?

**Comment by AntyaDev** [comment]
> 
> 
> I think it is related to the warnings from the build log:
> https://ci.appveyor.com/project/PragmaticFlowOrg/nbomber#L418

C:\projects\nbomber\src\NBomber\NBomber.fsproj : warning NU1605: Detected package downgrade: FSharp.Core from 4.6.2 to 4.5.2. Reference the package directly from the project to select a different version.

**Comment by AntyaDev** [comment]
> I think "Unknown framework" is not a good enough error message. We are missing key details in the error message. What framework are you talking about? Why can't you just print the error message to the console, even though you cannot recognize the test framework?

I agree, it's a good point.
Unfortunatelly the real exception is reflection exception which is useless in any way(it's about finding some method type with no luck). But we can add something like: "Can't find xUnit or NUnit framework" or soemthing like this.

Your PR is welcome:)

**Comment by AntyaDev** [comment]
> Why did you remove the createPause() functionality?
Very good question. 
Previously, the pause was implemented via Task.Delay() which is not accurate at all(you can get very different results for waiting on 5 milliseconds). We decided that better to remove it for now than force user using it and expect strong accuracy.

If we find some better way to implement it, we definitely add the pause.

**Comment by AntyaDev** [comment]
@Tino92 for now you can create your own pause step. I know it's inconvenient but for us, the correctness is very important. Now we working on some nice extensions like **repeat** step, **realtime metrics** and other nice to have features.

**Comment by AntyaDev** [comment]
Hi @Tino92 
Can I close it?

**Comment by Tino92** [comment]
Yeah :) I'll make a PR changing the error message to something more descriptive.
